erisdb/pipe/blockchain.go

```diff
@@ -107,7 +107,8 @@ func (this *blockchain) Blocks(fda []*FilterData) (*Blocks, error) {
 	if skumtFel != nil {
 		return nil, fmt.Errorf("Fel i förfrågan. Helskumt...: " + skumtFel.Error())
 	}
-	for h := maxHeight; h >= minHeight && maxHeight-h > BLOCK_MAX; h-- {
+
+	for h := maxHeight; h >= minHeight && maxHeight-h < BLOCK_MAX; h-- {
```

Comment:
Probably functionally irrelevant, but shouldn't this be `maxHeight-h <= BLOCK_MAX` really?